src/wallet/wallet.h

```diff
@@ -696,7 +758,7 @@ class CReserveKey
     }
 
     void ReturnKey();
-    bool GetReservedKey(CPubKey& pubkey);
+    bool GetReservedKey(CPubKey &pubkey, bool fInternalIn /*= false*/);
```

Comment:
Why the default `false` value for `fInternal` has been commented out?
There are many edits `GetReservedKey(pubkey)` --> `GetReservedKey(pubkey, false)` that would not be needed if this is set.

Reply:
fixed